### PR TITLE
BUILD.gn: Make Windows compilation work

### PR DESCRIPTION
**Pull request by Kangz**
PTAL, with this CL all Dawn code compiles on all platforms using GN.

### CODE REVIEW

File: src/dawn_native/RenderPassDescriptor.cpp [hunk 1]
```diff
@@ -29,7 +29,8 @@ namespace dawn_native {
           mDepthStencilAttachmentSet(builder->mDepthStencilAttachmentSet),
           mDepthStencilAttachment(builder->mDepthStencilAttachment),
           mWidth(builder->mWidth),
-          mHeight(builder->mHeight) {
+          mHeight(builder->mHeight),
+	      mDevice(builder->GetDevice()){
```
Comment (SenorBlanco) [inline src/dawn_native/RenderPassDescriptor.cpp]: weird formatting?

File: src/dawn_native/RenderPassDescriptor.h [hunk 2]
```diff
@@ -61,6 +61,8 @@ namespace dawn_native {
         uint32_t GetWidth() const;
         uint32_t GetHeight() const;
 
+		DeviceBase* GetDevice() const;
```
Comment (SenorBlanco) [inline src/dawn_native/RenderPassDescriptor.h]: Indentation seems wrong.

File: src/dawn_native/RenderPassDescriptor.h [hunk 3]
```diff
@@ -70,6 +72,8 @@ namespace dawn_native {
 
         uint32_t mWidth;
         uint32_t mHeight;
+
+		DeviceBase* mDevice;
```
Comment (SenorBlanco) [inline src/dawn_native/RenderPassDescriptor.h]: Same here.

File: src/dawn_native/RenderPassDescriptor.cpp (hunk 1 above)
Comment (Kangz) [reply src/dawn_native/RenderPassDescriptor.cpp]: Done

File: src/dawn_native/RenderPassDescriptor.h (hunk 2 above)
Comment (Kangz) [reply src/dawn_native/RenderPassDescriptor.h]: Done

File: src/dawn_native/RenderPassDescriptor.h (hunk 3 above)
Comment (Kangz) [reply src/dawn_native/RenderPassDescriptor.h]: Done

**Comment by Kangz** [comment]
Landing now, TBR @kainino0x

**Pull request merged**